src/InstrumentationEngine.Attach/Program.cs

```diff
@@ -54,6 +60,57 @@ internal static int Main(string[] args)
                 return ExitCodeFailure;
             }
 
+            // Attempt to resolve fullpath
+            string configFilePath = Path.GetFullPath(args[1]);
+            if (!File.Exists(configFilePath))
+            {
+                WriteError(Invariant($"Could not find file '{configFilePath}'."));
+                return ExitCodeFailure;
+            }
+
+            #endregion
+
+            #region Parse Configuration Sources
+            InstrumentationConfigurationSources? configSources = null;
+
+            // Schema file embedded name is the same that the full type name that was generated from the schema file
+            string schemaResourceName = typeof(InstrumentationConfigurationSources).FullName + ".xsd";
+
+            // Read schema file
+            using (Stream? schemaStream = Assembly.GetExecutingAssembly().GetManifestResourceStream(schemaResourceName))
+            using (XmlReader schemaReader = XmlReader.Create(schemaStream, new XmlReaderSettings() { XmlResolver = null }))
+            {
+                // Create reader settings that validate description file
```

Comment:
>description [](start = 56, length = 11)

Updating comments

Reply:
I pulled these from VS as there were tons of XMLResolver bugs that were fixed.

---
In reply to: [388506215](https://github.com/microsoft/CLRInstrumentationEngine/pull/220#discussion_r388506215) [](ancestors = 388506215)